pyscf/adc/radc.py

```diff
@@ -1697,6 +1701,35 @@ def get_trans_moments(adc):
     T = np.array(T)
     return T
 
+def density_matrix(adc, T):
```

Comment:
There are several problems here:
1) One should be able to run this function directly from ground-state ADC class, so there should be a ground-state ADC class member that points to this function
2) This function can take T as an argument, but not as a required one. It should be a conditional argument. If T is not provided, this code should call a function for calculating T using IP-ADC of the corresponding approximation (e.g., ADC(2) or (3), etc). 
3) The naming of the function should be consistent with names of functions in PySCF for calculating density matrices. The convention is that spin-adapted 1-RDM (alpha + beta) is computed using make_rdm1() and spin-orbital (alpha and beta separately) is computed as make_rdm1s(). It is OK to keep this function's name as density_matrix outside of the class, but inside the ground-state RDM class it should be named appropriately. For RADC we can have two functions: make_rdm1 returning identical total 1-RDM and make_rdm1s returning identical alpha and beta components. Similarly, for UADC only make_rdm1s should be available returning alpha and beta separately